python/src/aiconfig/eval/README.md

```diff
@@ -0,0 +1,110 @@
+# Tl;Dr
+
+Iterate on AIConfig faster. Measure AIConfig quality improvements and catch regressions. This module is closely related to [Promptfoo](https://www.promptfoo.dev/) and in fact it provides the option to integrate directly with Promptfoo. Please refer to their docs for motivating examples and usage details as needed.
+
+- You specify an AIConfig, test data, and evaluation metrics / assertions.
+- Each test case is a pair of (data, evaluation_metric).
+- Metrics can be anything numerical (float/int) or any boolean condition (pass/fail).
+- AIConfig eval library runs them through your AIConfig and presents you a table to compare the results. Example:
+
+Inputs:
+
+```
+test_inputs = [
+    "Food",
+    "Iconic midtown skyscrapers",
+]
+expected_substrings = [
+    "Magnolia Bakery",
+    "Empire State Building",
+]
```

Comment:
This is different format from L123 where the input and output are tupled together

Reply:
You are correct, I need to update this.